optimas/diagnostics/exploration_diagnostics.py

```diff
@@ -100,7 +106,30 @@ def _create_exploration(
             ),
             evaluator=Evaluator(sim_function=None),
             history=history_path,
+            exploration_dir_path=exploration_dir_path,
+        )
+
+    def _create_sim_dir_paths(self):
```

Comment:
Just a comment:
This creates a separate dictionary with the simulation directory paths with the trial indexes as keys.
An alternative solution could be to add the simulation paths as a new column to the history DataFrame (adding `None` when not present).